numpy/core/tests/test_deprecations.py

```diff
@@ -646,6 +646,23 @@ def foo():
         test_case_instance.assert_deprecated(foo)
         test_case_instance.tearDown()
 
+class TestClassicIntDivision(_DeprecationTestCase):
+    """
+    See #7949. Deprecate the numeric-style dtypes with -3 flag in python 2 
+    if used for division
+    List of data types: http://docs.scipy.org/doc/numpy/user/basics.types.html
+    """
+    def test_int_dtypes(self):
+        deprecated_types = [
+           'bool_', 'int_', 'intc', 'intp', 'int8', 'int16', 'int32', 'int64',
+           'uint8', 'uint16', 'uint32', 'uint64'
+            ]
+        if sys.version_info[0] < 3 and sys.py3kwarning:
+            import operator as op
+            for dt in deprecated_types:
+                a = np.array([1,2,3], dtype=dt)    
```

Comment:
PEP8 discourages extraneous whitespace immediately before comma,semicolon (see second bullet in section Pet Peeves)
In this case, space is after comma which should be acceptable since it avoids cluttering (IMHO)

Please correct me if I am wrong. For consistency and code quality, I am ready to change my coding habits :)